cmd/virt-handler/Dockerfile

```diff
@@ -20,9 +20,18 @@ FROM fedora:26
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
-RUN dnf -y install libvirt-client \
+RUN dnf -y install libvirt-client cyrus-sasl-scram \
```

Comment:
So, just invastigated scram and it looks like it only does sha1, which is weak.  Is there another option?